_sass/frogtek-theme.scss

```diff
@@ -420,6 +420,10 @@ footer {
   text-transform: lowercase;
 }
 
+.center-div {
```

Comment:
This is widely considered a very bad practice... You shouldn't name Css classes related to the actual style they apply.